### PR TITLE
Initial commit of json schema for patient model

### PR DESCRIPTION
**Pull request by mlos100**
Add json schema files for patient model to resources/model_json_schema

### CODE REVIEW

File: resources/model_json_schema/cda_identifier.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  "definitions": {
+    "CDAIdentifier" : {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "root": {
+          "type": ["string", "null"],
+          "minLength": 0,
+          "title": "root schema.",
```
Comment (eedrummer) [inline resources/model_json_schema/cda_identifier.json]: Trailing comma should be removed

File: resources/model_json_schema/cda_identifier.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        },
+        "extension": {
+          "type": ["string", "null"],
+          "minLength": 0,
+          "title": "extension schema.",
```
Comment (eedrummer) [inline resources/model_json_schema/cda_identifier.json]: Trailing comma should be removed

File: resources/model_json_schema/entry.json [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        { "$ref": "thingWithCodes.json#/definitions/ThingWithCodes"},
+        { "properties": {
+          "_id": {
+            "type": "string",
+            "minLength": 1,
+            "title": "_id schema.",
+            "description": "An explanation about the purpose of this instance described by this schema."
+          },
+          "cda_identifier": {
+            "$ref": "cdaIdentifier.json#/definitions/CDAIdentifier",
```
Comment (eedrummer) [inline resources/model_json_schema/entry.json:21]: Broken reference. Should be cda_identifier.

File: resources/model_json_schema/provider.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+              "$ref": "organization.json#/definitions/Organization"
+            },
+    
+            "cda_identifiers": {
+              "type": "array",
+              "minItems": 0,
+              "uniqueItems": true,
+              "additionalItems": true,
+              "items": {
+                "$ref": "cdaIdentifier.json#/definitions/CDAIdentifier"
```
Comment (eedrummer) [inline resources/model_json_schema/provider.json:32]: Broken reference

Comment (mlos100) [inline resources/model_json_schema/provider.json:32]: Changed the file to match the convention used elsewhere: cda_identifier.json => cdaIdentifier.json

File: resources/model_json_schema/cda_identifier.json (hunk 2 above)
Comment (mlos100) [inline resources/model_json_schema/cda_identifier.json]: removed the 2 invalid commas

**Comment by eedrummer** [comment]
Rebased and merged

**Pull request closed without merging**